nf_core/pipeline-template/README.md

```diff
@@ -55,7 +55,11 @@ On release, automated continuous integration tests run the pipeline on a full-si
     <!-- TODO nf-core: Update the example "typical command" below used to run the pipeline -->
 
     ```console
-    nextflow run {{ name }} -profile <docker/singularity/podman/shifter/charliecloud/conda/institute> --input samplesheet.csv --genome GRCh37
+    nextflow run {{ name }} \
```

Comment:
Maybe we revert this change because in Point 3 above it is a single line. We can always change it at the pipeline leve.